xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/importinline.vm

```diff
@@ -35,171 +35,177 @@
-               <label for="xwikiuploadfile" class="hidden">$services.localization.render('core.viewers.attachments.upload.file')</label>
-               <input id="xwikiuploadfile" type="file" name="filepath" value="" size="30" class="uploadFileInput"/>
-               <span class="buttonwrapper">
-                 <input type="submit" value="Upload" class="button" />
-               </span>
-            </div>
-          </fieldset>
-        </div>
-      </form>
-    <div class="legend">$services.localization.render('core.importer.availablePackages')</div>
-    <div id="packagelistcontainer">
-      #template('packagelist.vm')
-    </div> ## packagelistcontainer
-    </div> ## packagelist
-    <div id="packagecontainer">
-      #if("$!request.file" != '')
-        ##
-        ## If we get here, it means javascript is not activated on the user's browser (otherwise the attachment link
-        ## click would have been intercepted). We display a basic UI (list of checkboxes) to chose which documents in the archive to import.
-        ##
-        #set($attach = $doc.getAttachment($request.file))
-        #if($attach)
-          #set($package = $services.xar.getXarPackage($attach.contentInputStream, true))
-          #if(!$package)
-            #error("There was an error reading the file ${escapetool.xml(${request.file})}. $!xcontext.import_error")
-          #else
-          <div class="legend">$services.localization.render('core.importer.availableDocuments')</div>
-          <div id="packageDescription">
-            <div class="packageinfos">
-              <div>
-                <span class="label">$services.localization.render('core.importer.package')</span>
-                <span class="filename">$attach.filename</span>
-              </div>
-              #if("$!package.packageName" != '')
-              <div>
-                <span class="label">$services.localization.render('core.importer.package.description')</span>
-                <span class="name">$package.packageName</span>
-              </div>
-              #end
-              #if("$!package.packageVersion" != '')
-              <div>
-                <span class="label">$services.localization.render('core.importer.package.version')</span>
-                <span class="version">$package.packageVersion</span>
-              </div>
-              #end
-              #if("$!package.packageAuthor" != '')
-              <div>
-                <span class="label">$services.localization.render('core.importer.package.author')</span>
-                <span class="author">$package.packageAuthor</span>
-              </div>
-              #end
-              #if("$!package.packageLicense" != '')
-              <div>
-                <span class="label">$services.localization.render('core.importer.package.licence')</span>
-                <span class="licence">$package.packageLicense</span>
-              </div>
-              #end
+  <div class="row">
```

Comment:
Instead of adding a new container, a better solution would be add the class to div#import.